src/seedsigner/views/view.py

```diff
@@ -243,3 +243,50 @@ def run(self):
         ).display()
         
         return Destination(MainMenuView, clear_history=True)
+
+class MicroSDToastView(View):
+    def event_handler(self, action, mount_dir):
```

Comment:
Could use a comment here explaining how/when this is called since a normal `View` doesn't operate this way. That being said, the `MicroSD` class doesn't need `ui_handler` flexibility; we're not going to be plugging in any other handler implementations there. So it may as well just have direct knowledge of the `MicroSDToastView`.

So rather than being passed in an instance, `MicroSD` could just instantiate this `View` and pass in the `action` and `mount_dir` as normal `view_args` and then explicitly call the `run()`.

So instead of:
```
if self.ui_handler:
    self.ui_handler(action, mount_dir)
```

Couldn't we just do:
```
toast_view = MicroSDToastView(action=foo, mount_dir=bar)
toast_view.run()
```

This way we keep the handler `View` identical to any other (maybe add a note at the end why we don't need to return a `Destination`). And it's a little clearer in the `MicroSD` loop that the toast view's `run()` will block.